src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -123,138 +135,194 @@ const UserEnrollmentForm = ({
+                                );
+                              }}
+                            >
+                              Terms and Conditions
+                            </Hyperlink>
+                            &nbsp;for Students
+                          </span>
+                        </div>
+                        {errors.studentTermsAndConditions && (
+                          <Form.Control.Feedback type="invalid">
+                            {errors.studentTermsAndConditions}
+                          </Form.Control.Feedback>
+                        )}
+                      </Form.Group>
+                    </Col>
+                  </Row>
+                  <Row className="mb-4">
+                    <Col xs={12} lg={8}>
+                      <p className="small">
+                        *Required
+                      </p>
+                    </Col>
+                  </Row>
+                  {!isAgeValid && (
+                    <Row className="mb-4 age-error-message">
+                      <Col xs={12} lg={12}>
+                        <p className="font-italic">
+                          Unfortunately you don&apos;t meet minimum age requirement and without any parent or guardian
+                          consent you can not proceed with your registration.
+                        </p>
+                      </Col>
+                    </Row>
                   )}
-                </Form.Group>
-              </Col>
-            </Row>
+                </Card.Section>
+              </Card.Body>
+            </Card>
+
             <Row>
-              <Col>
-                <Form.Group>
-                  <div className="d-flex align-items-center">
-                    <CheckboxControl
-                      className="flex-shrink-0"
-                      checked={values.studentTermsAndConditions}
-                      name="studentTermsAndConditions"
-                      onChange={handleChange}
-                      onBlur={handleBlur}
-                      aria-label="I agree to GetSmarter's Terms and Conditions for Students"
-                    />
-                    <span aria-hidden>
-                      I agree to GetSmarter&apos;s{' '}
-                      <Hyperlink
-                        destination={config.GETSMARTER_STUDENT_TC_URL}
-                        target="_blank"
-                        onClick={() => {
-                          sendEnterpriseTrackEvent(
-                            enterpriseId,
-                            'edx.ui.enterprise.learner_portal.executive_education.checkout_form.student_terms_conditions.clicked',
-                          );
+              <Col className="justify-content-end">
+                <Row className="justify-content-end">
+                  {
+                    isAgeValid && (
+                      <StatefulButton
```

Comment:
The `StatefulButton` is not properly aligned with the rest of the page/form:

<img width="1246" alt="image" src="https://user-images.githubusercontent.com/2828721/231528156-4f18b27a-8286-4dc1-9fea-0e56be215000.png">

Note how it extends beyond the width of the "Registration summary" card. I would have expected the button to be flush with the rightmost edge of the "Registration summary" card for consistent alignment.